collection.go

```diff
@@ -88,6 +88,43 @@ func (c *LoggerCollection) getLogger(name string, level LogLevel,
 	return logger
 }
 
+// ConfigureLoggers configures loggers according to the given string
+// specification, which specifies a set of loggers and their associated
+// logging levels.  Loggers are semicolon-separated; each
+// configuration is specified as <logger>=<level>.  White space outside of
+// logger names and levels is ignored.  The default level is specified
+// with the name "DEFAULT".
+//
+// An example specification:
+//	`<root>=ERROR; foo.bar=WARNING`
+func (c *LoggerCollection) ConfigureLoggers(specification string) error {
+	confs := strings.Split(strings.TrimSpace(specification), ";")
+	for i := range confs {
+		conf := strings.SplitN(strings.TrimSpace(confs[i]), "=", 2)
```

Comment:
it feels like the trim spaces are weird. this could still result in conf[0] ending in spaces

Reply:
I guess technically a logger could have spaces in the name. I should remove all of the trims.